ui/components/app/import-token/token-list/token-list.component.js

```diff
@@ -74,7 +74,7 @@ export default class TokenList extends Component {
             display={Display.Flex}
             flexDirection={FlexDirection.Column}
           >
-            {Array(12)
+            {Array(Math.min(12, results.length))
```

Comment:
We clamp the number of items we are processing.

This is because our results we get back might be less than 12.